### DISCUSSION TITLE
Port 80 unraid server

### DISCUSSION THREAD

**Discussion by Dimtar** [nextcloud/all-in-one#817] [Questions] [answered]
Hi all.

Trying to get NC-AIO running on an unRaid box.
All the containers have started except the Apache container.
I believe this is because the unRaid OS needs port 80 already:
![image](https://user-images.githubusercontent.com/3172977/173229377-3c587711-8f68-452f-b4a9-7664c4180240.png)

Is there a way to tell the system to use a host port other then port 80?

**Comment by szaimen** [comment, marked as answer]
Hi, please follow https://github.com/nextcloud/all-in-one/blob/main/reverse-proxy.md

**Reply by Dimtar** [reply]
I did follow that guide, NPM is ready to go. This is the docker container that was created:
![image](https://user-images.githubusercontent.com/3172977/173281859-c3088594-03ee-4183-8c21-487335ceec1d.png)
I believe the container cannot start because the unraid OS already uses port 80

The apache container is the only container that won't start.

**Reply by szaimen** [reply]
if you provide `APACHE_PORT=11000` via environmental variable to the mastercontainer correctly, the apache containerit should only use this port and not port 80. Can you show me the docker run command that you used?

**Reply by Dimtar** [reply]
This is what unRaid lists for the run command:
`/usr/local/emhttp/plugins/dynamix.docker.manager/scripts/docker run -d --name='nextcloud-aio-mastercontainer' --net='bridge' -e TZ="Australia/Sydney" -e HOST_OS="Unraid" -e HOST_HOSTNAME="ASTRO" -e HOST_CONTAINERNAME="nextcloud-aio-mastercontainer" -e 'APACHE_PORT'='11000' -e 'NEXTCLOUD_DATADIR'='/mnt/user/ncdata' -l net.unraid.docker.managed=dockerman -l net.unraid.docker.webui='http://[IP]' -l net.unraid.docker.icon='https://github.com/Muwahhidun/logo/raw/main/nextcloud-aio-logo.png' -p '8080:8080/tcp' -v '/mnt/user/ncaio':'/mnt/docker-aio-config':'rw' -v '/var/run/docker.sock':'/var/run/docker.sock':'ro' 'nextcloud/all-in-one:latest' `

**Reply by szaimen** [reply]
yes, looks good to me.

In general it should work fine:

https://myunraid-ru.translate.goog/nextcloud-aio/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=de&_x_tr_pto=wapp

**Reply by Dimtar** [reply]
Reading the guide supplied I think the issue is probably the permissions on the ncdata folder. I will keep working on it.
